src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationLambdaInEnum2.java

```diff
@@ -0,0 +1,68 @@
+/* Config:                                                                          //indent:0 exp:0
+ * This test-input is intended to be checked using following configuration:         //indent:1 exp:1
+ * tabWidth = 8                                                                     //indent:1 exp:1
+ * basicOffset = 4                                                                  //indent:1 exp:1
+ * braceAdjustment = 0                                                              //indent:1 exp:1
+ * caseIndent = 4                                                                   //indent:1 exp:1
+ * throwsIndent = 4                                                                 //indent:1 exp:1
+ * forceStrictCondition = false                                                     //indent:1 exp:1
+ * lineWrappingIndentation = 4                                                      //indent:1 exp:1
+ */                                                                                 //indent:1 exp:1
+
+package com.puppycrawl.tools.checkstyle.checks.indentation.indentation; //indent:0 exp:0
+
+import java.util.function.Function; //indent:0 exp:0
+
+enum InputIndentationLambdaInEnum2 { //indent:0 exp:0
+	enum1( //indent:8 exp:8
+		v -> v, //indent:16 exp:16
+		new Object() //indent:16 exp:16
+	), //indent:8 exp:8
+
+	enum2( //indent:8 exp:8
+	v //indent:8 exp:8
+		-> v, //indent:16 exp:16
+		new Object() //indent:16 exp:16
+	), //indent:8 exp:8
+
+	enum3( //indent:8 exp:8
+v -> v, //indent:0 exp:4 warn
+		new Object() //indent:16 exp:16
+	), //indent:8 exp:8
+
+	enum4( //indent:8 exp:8
+	v -> //indent:8 exp:8
+v, //indent:0 exp:0
```

Comment:
So, check is ok with any level of indentation for this parameter?

Reply:
similar 🤔 https://github.com/checkstyle/checkstyle/issues/4070

Reply:
I think this may be related to https://github.com/checkstyle/checkstyle/issues/8965, as this issue also effects parameters/ arguments. Please add comment in above issue with minimal example of this code, as we do for bug reports.

Reply:
example added

Reply:
Resolved.